lib/daru/io/exporters/csv.rb

```diff
@@ -5,16 +5,28 @@ module IO
     module Exporters
       module CSV
```

Comment:
I agree with lazy call. Lets go ahead with this. You can use your buffer period to port it library-wide.